WMF Framework/Remote Notifications/Model/RemoteNotification+CoreDataProperties.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import CoreData
+
+extension RemoteNotification {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<RemoteNotification> {
+        return NSFetchRequest<RemoteNotification>(entityName: "RemoteNotification")
+    }
+
+    @NSManaged public var agent: String?
+    @NSManaged public var affectedPageID: String?
+    @NSManaged public var categoryString: String?
+    @NSManaged public var date: NSDate?
+    @NSManaged public var id: String?
+    @NSManaged public var message: String?
+    @NSManaged public var typeString: String?
+    @NSManaged public var wiki: String?
+    @NSManaged public var wasRead: Bool
+    @NSManaged public var wasSeen: Bool
+    @NSManaged public var isExcluded: Bool
```

Comment:
would it make sense to combine these three booleans into a state enum?

Reply:
yes!